### PR TITLE
[Merged by Bors] - feat: port/CategoryTheory.Limits.Yoneda

### PR DESCRIPTION
**Pull request by mattrobball**
---

- [x] depends on: #2380
[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by semorrison** [comment]
This PR/issue depends on:
* leanprover-community/mathlib4#2380
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

**Comment by semorrison** [comment]
This PR/issue depends on:
* ~~leanprover-community/mathlib4#2380~~
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

File: Mathlib/CategoryTheory/Limits/Yoneda.lean [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+open Limits
+
+/--n The yoneda embedding `yoneda.obj X : Cᵒᵖ ⥤ Type v` for `X : C` preserves limits. -/
+instance yonedaPreservesLimits (X : C) : PreservesLimits (yoneda.obj X)
+    where preservesLimitsOfShape {J} 𝒥 :=
+    {
+      preservesLimit := fun {K} =>
+        {
+          preserves := fun {c} t =>
```
Comment (jcommelin) [inline Mathlib/CategoryTheory/Limits/Yoneda.lean:91]:
```suggestion
    { preservesLimit := fun {K} =>
        { preserves := fun {c} t =>
```

File: Mathlib/CategoryTheory/Limits/Yoneda.lean [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+                  · exact Quiver.Hom.unop_inj (congrFun (w j) x) } } }
+#align category_theory.yoneda_preserves_limits CategoryTheory.yonedaPreservesLimits
+
+/-- The coyoneda embedding `coyoneda.obj X : C ⥤ Type v` for `X : Cᵒᵖ` preserves limits. -/
+instance coyonedaPreservesLimits (X : Cᵒᵖ) : PreservesLimits (coyoneda.obj X)
+    where preservesLimitsOfShape {J} 𝒥 :=
+    {
+      preservesLimit := fun {K} =>
+        {
+          preserves := fun {c} t =>
```
Comment (jcommelin) [inline Mathlib/CategoryTheory/Limits/Yoneda.lean:111]:
```suggestion
    { preservesLimit := fun {K} =>
        { preserves := fun {c} t =>
```

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**